ironfish/src/rpc/routes/transactions/sendTransaction.ts

```diff
@@ -98,33 +102,41 @@ router.register<typeof SendTransactionRequestSchema, SendTransactionResponse>(
         publicAddress: receive.publicAddress,
         amount: CurrencyUtils.decode(receive.amount),
         memo: receive.memo,
-        assetIdentifier: Asset.nativeIdentifier(),
+        assetIdentifier: Buffer.from(receive.assetIdentifier, 'hex'),
       }
     })
 
     const fee = CurrencyUtils.decode(transaction.fee)
-    const sum = receives.reduce((m, c) => m + c.amount, fee)
```

Comment:
This line sets the initial value to the fee. I think we're missing the fee being added to the native asset amount in the new version